editor/plugins/script_editor_plugin.cpp

```diff
@@ -3611,8 +3611,8 @@ void ScriptEditor::clear_docs_from_script(const Ref<Script> &p_script) {
 	Vector<DocData::ClassDoc> documentations = p_script->get_documentation();
 	for (int j = 0; j < documentations.size(); j++) {
 		const DocData::ClassDoc &doc = documentations.get(j);
-		if (EditorHelp::get_doc_data()->has_doc(doc.name)) {
-			EditorHelp::get_doc_data()->remove_doc(doc.name);
+		if (EditorHelp::has_doc(doc.name)) {
```

Comment:
Not need to call `EditorHelp::has_doc` because it's done in `EditorHelp::remove_doc`.